cfg/system.config.in

```diff
@@ -117,3 +118,17 @@ ffi-lib-dir       = @FFILibDir@
 #=======================
 
 with-libdw = @UseLibdw@
+
+# Installation:
+#=======================
+
+prefix          = @prefix@
```

Comment:
Is `prefix` used only for installation? If yes, I guess `install-prefix` would be easier to understand.